qrisk/tests/test_stats.py

```diff
@@ -40,32 +40,90 @@ class TestStats(TestCase):
             '2000-1-1',
             periods=500,
             freq='D'))
-    px_list = np.array(
-        [10, -10, 10]) / 100.  # Ends in drawdown
-    dt = pd.date_range('2000-1-3', periods=3, freq='D')
 
-    px_list_2 = [1.0, 1.2, 1.0, 0.8, 0.7, 0.8, 0.8, 0.8]
-    dt_2 = pd.date_range('2000-1-3', periods=8, freq='D')
+    s_1 = pd.Series(np.array([10, -10, 10]) / 100.,
+                    index=pd.date_range('2000-1-3', periods=3, freq='D'))
+
+    s_2 = pd.Series([1.0, 1.2, 1.0, 0.8, 0.7, 0.8, 0.8, 0.8],
+                    index=pd.date_range('2000-1-3', periods=8, freq='D'))
+
+    s_3 = pd.Series(np.array([1, 10, -2, 2, 3, 2, 1, -10])/100,
+                    index=pd.date_range('2000-1-3', periods=8, freq='D'))
+
+    one = [-0.00171614, 0.01322056, 0.03063862, -0.01422057, -0.00489779,
+           0.01268925, -0.03357711, 0.01797036]
+    two = [0.01846232, 0.00793951, -0.01448395, 0.00422537, -0.00339611,
+           0.03756813, 0.0151531, 0.03549769]
+    df_index_simple = pd.date_range('2000-1-3', periods=8, freq='D')
+    df_index_week = pd.date_range('2000-1-3', periods=8, freq='W')
+    df_index_month = pd.date_range('2000-1-3', periods=8, freq='M')
+
+    d_simple = {'one': pd.Series(one, index=df_index_simple),
+                'two': pd.Series(two, index=df_index_simple)}
+    df_simple = pd.DataFrame(d_simple)
+
+    d_week = {'one': pd.Series(one, index=df_index_week),
+              'two': pd.Series(two, index=df_index_week)}
+    df_week = pd.DataFrame(d_week)
+
+    d_month = {'one': pd.Series(one, index=df_index_month),
+               'two': pd.Series(two, index=df_index_month)}
+    df_month = pd.DataFrame(d_month)
+
+    @parameterized.expand([
+        (simple_rets, 0, 0.33100000000000041),
```

Comment:
since we are doing almost equal should we remove the insignificant digits in these expecteations. They seem to add noise to the expected value.